hudi-client/hudi-client-common/src/main/java/org/apache/hudi/io/FileGroupReaderBasedAppendHandle.java

```diff
@@ -114,6 +115,7 @@ public List<WriteStatus> close() {
       if (writeStatus.getStat().getRuntimeStats() != null) {
         writeStatus.getStat().getRuntimeStats().setTotalScanTime(readStats.getTotalLogReadTimeMs());
       }
+      writeStatus.getStat().setPrevCommit(operation.getBaseInstantTime());
```

Comment:
Actually the `prevCommit` represents the base instant time or "version" of the file slice, so this is correct.

Reply:
it could be the first delta commit time if there is no base file, for log compaction, this seems incorrect, current only cdc read relies on the prevCommit for resolving dependencies, we might ignore it for log compaction because query base instant time per file-split is very costly.

Reply:
This will match the behavior of log-compaction before 1.0.0. Was it intentional to change this portion of the commit metadata for 1.0.0?